docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/anemia_impairment/index.rst

```diff
@@ -7,3 +7,173 @@ Anemia Impairment
+
+The following table displays the cause- and sex-specific hemoglobin shifts used for the anemia causal attribution process in GBD 2019. These shifts are hosted `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia_causal_attribution/browse/priors/hb_shifts.csv>`_.
+
+.. csv-table:: Cause- and Sex-Specific Hemoglobin Shifts
+  :widths: 15 15 15
+  :file: hb_shifts.csv
+
+Iron Responsive Causes
+^^^^^^^^^^^^^^^^^^^^^^
+
+A list of which causes of anemia are iron responsive can be found in the excel sheet hosted `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia_causal_attribution/browse/in_out_meid_map.xlsx>`_.
+
+Vivarium Modeling Strategy
+--------------------------
+
+Scope
+++++++++
+
+The Vivarium modeling strategy for the anemia impairment will first rely on the :ref:`Hemoglobin Distribution Model <2019_hemoglobin_distribution>`. Vivarium simulants should first be assigned a hemoglobin value and then their anemia status can be evaluated by the hemoglobin thresholds shown on this page. YLDs should be accrued according to the severity-specific disability weights listed in this document. Specific causes (or groups of causes, for example iron-responsive causes) of anemia can also be assigned to individual if relevant/necessary for the project.
+
+Restrictions
+++++++++++++
+
+.. list-table:: GBD 2019 Anemia Impairment Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     - Note the pregnancy adjustment for women of reproductive age
+   * - Age group start
+     - Birth (age_group_id=2)
+     - Note early and late neonatal age group (ID 2,3) modelling exception
+   * - Age group end
+     - 95+
+     - 
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+The assumptions and limitations of this vivarium anemia impairment model include the same assumptions and limitations as the :ref:`Hemoglobin distribution model <2019_hemoglobin_model>`.
+
+Additionally, the GBD 2019 causal attribution process assumes no change in hemoglobin standard deviation by cause, which is likely not accurate, but represents a significant data gap. The causal attribution process also relies on residual attribution rather than direct attribution to several causes of anemia, including dietary iron deficiency anemia, which makes the prevalence estimates of these causes sensitive to the prevalence estimates of other anemia causes. 
+
+Validation Criteria
++++++++++++++++++++
+
+Prevalence of severity-specific anemia should be approximately equal to the GBD 2019 severity-specific anemia impairment prevalence.
```

Comment:
I don't understand the difference between the two things being compared here.

Reply:
I made edits based on all of you above comments and also tried to clarify what is meant by these two descriptions here in the latest commits. Let me know if you still have any questions/uncertainties. Thanks for your review!

Reply:
Looks good!